### PR TITLE
Header component

### PR DESCRIPTION
**Pull request by AymanAlqoqa**
- Create Header Component for the logged & not Logged member 
- add styling for it.

how to use:
- `git fetch origin header-component`
- `git checkout header-component`
- `cd client & npm i`
- `cd ..`
- `npm run dev`
Header component needs prop name isLogged which is boolean.
Header component checks the local storage for userInfo.
thanks

relates #13

### CODE REVIEW

File: client/src/Components/Header/index.jsx [hunk 1]
```diff
@@ -1,9 +1,130 @@
...
+
+  componentDidMount() {
+    // const userInfo = localStorage.getItem('userInfo');
+    // this.setState({ userInfo });
+  }
+
+  render() {
+    const { islogged } = this.props;
+    // default values
+    const { fullName, username, avatar } = this.state.userInfo;
```
Comment (Fatmasiam) [inline client/src/Components/Header/index.jsx]:
```javascript
  const {
      userInfo: { fullName, username, avatar },
    } = this.state;

```
I think will solve the eslint problem

File: client/src/Components/Header/Notification/index.jsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import { Dropdown } from 'react-bootstrap';
+import './style.css';
+
+class Notification extends Component {
+  // this notification will fetched from back-end socket
+  state = {
```
Comment (Angham116) [inline client/src/Components/Header/Notification/index.jsx]: to be more clearly, add utilise file for notifications and import it here then in the ```componentDidMount``` function update the state with notification details.
this way will facilitate for us when joining the backend with front-end

File: client/src/Components/Header/index.jsx (hunk 1 above)
Comment (AymanAlqoqa) [reply client/src/Components/Header/index.jsx]: we will change it when we use socket
thanks

File: client/src/Components/Header/Notification/index.jsx (hunk 2 above)
Comment (AymanAlqoqa) [reply client/src/Components/Header/Notification/index.jsx]: I think, we will change it when we use actual data from socket
thanks

**Pull request merged**